src/main/java/med/volkl/api/medico/Medico.java

```diff
@@ -7,7 +7,6 @@
 import lombok.NoArgsConstructor;
 import med.volkl.api.endereco.*;
 
-
 @Table(name = "medicos")
 @Entity(name = "Medico")
```

Comment:
se a entidade for o mesmo nome da classe, não precisa usar isso
```suggestion
@Entity
```

Reply:
Linha 41:48 depois estuda `MapStruct`